### DISCUSSION TITLE
Apa Bedanya AdBlockID dengan AdBlocID Plus? Mungkinkah digabung?

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1333] [Ideas]
Tolong dong ini saya punya usul agar adblockid plus itu di jadikan satu saja dengan adblockid
Jadi yg subcribe gak harus tambah subscribe lagi ke adblockid plus
Mungkin sebagian masih gak tau fungsi adblockid plus, jadi sebagian orang masih
Menggunakan tampermonkey dan anti adblock killer dan sebagainya🙏🙏🙏

**Comment by 3xploiton3** [comment]
https://github.com/realodix/AdBlockID/blob/master/docs/comparison_table.md

@realodix tabel penjelasan sepertinya perlu diperjelas lagi
semisal fitur anti adblock killer apakah termasuk di adblockid atau adblockid plus
+ screenshot contoh menggunakan adblockid plus dengan tidak menggunakan 
+ berat (sized) file adblockid plus

**Reply by realodix** [reply]
Nanti Saya akan update, karena Saya sendiri juga bingung untuk menulis penjelasannya. Penjelasakan singkatnya akan Saya jelaskan di bawah.

**Comment by Jokopentil** [comment]
Begini loh, maksud saya di sebagian orang untuk menangani anti adblock
Itu memakai anti adblock killer dan tampermonkey
Sedangkan ada juga kemungkinan
Adanya adblockid plus
Jika adblockid dan adblockid plus di jadikan satu
Kita tidak perlu lagi tampermonkey atau
Subscribe tambahan untuk filter adblockid plus
Jika kita menggunaka adguard tentu hanya perlu menyalakan filter adblockid yg sudah ada 
Tentu sangat praktis sekali
Tapi untuk pengguna adblock plus dan adblock geta, filter ini gak berfungsi
Tapi untuk pengguna ublock, adblockid plus sangat lancar![20230205_195607_rmscr.jpg](https://user-images.githubusercontent.com/114223791/216819972-fb05c427-338f-474b-8b65-f8c290b6dd74.jpg)

Seperti contoh dalam gambar
Situs layardrama.icu
Ini walaupun pakai tamperkey gak bisa kebuka jika pakai adblock plus
Kesimpulannya ini hanya untuk agar lebih efisien lagi jadi antara adblockid dan adblockid jadi lebih ringkas🙏🙏🙏

**Comment by Jokopentil** [comment]
Maaf ralat antara adblockid dan adblockid plus di jadikan satu filter

**Comment by Jokopentil** [comment]
![20230205_200923_rmscr.jpg](https://user-images.githubusercontent.com/114223791/216821160-44a15c6b-a904-4e6f-8cb4-3f2f5396403e.jpg)

![20230205_201440_rmscr.jpg](https://user-images.githubusercontent.com/114223791/216821171-5ad27ab3-c405-4465-aec3-8f3dd850ce87.jpg)

Nah kalo lebih ringkas dia gak perlu lagi costum subscrib ke adblockid plus

**Comment by 3xploiton3** [comment]
tidak perlu digabung, karena peruntukannya masing2 itu opsional dari pengguna,
klo ada pop-up anti-adblock yg blm masuk dalam filter laporkan aja di sini,

**Comment by Jokopentil** [comment]
Ya itu sekedar usulan mungkin bagi dev
Datanya jadi banyak
Bagi sebagian orang kan taunya adblockid yag sudah terinstal di adguard
Ya klo usulan belum bisa diterma ya sudah

**Comment by realodix** [comment]
Halo Gan

Saya masih bingung sebenarnya tentang ide apa yang Anda usulkan?

Dari yang Anda jabarkan, Saya hanya menangkap 1 point saja:
- AdBlockID digabungkan jadi satu dengan AdBlockID Plus

**Jawaban**
Pada dasarnya yang membedakannya adalah hanya di batasan/cakupan bahasa saja.
- AdBlockID, sesuai penjelasannya di halaman depan, ini merupakan filter yang melengkapi filter internasional untuk menghalau iklan di situs Bahasa Indonesia. Batasannya jelas, hanya untuk situs berbahasa Indonesia Saja.
- AdBlockID Plus, pada dasarnya ini adalah personal filter yang Saya bagikan. Karena ini adalah personal filter, maka tidak ada batasan bahasa di sini. Biasanya Saya isi dengan situs berbahasa inggris yang sering Saya kunjungi.

Jadi tidak mungkin menyatukan AdBlockID dan AdBlockID Plus

---
from https://github.com/realodix/AdBlockID/discussions/1333#discussioncomment-4879070
> Jika kita menggunaka adguard tentu hanya perlu menyalakan filter adblockid yg sudah ada
> Tentu sangat praktis sekali
> Tapi untuk pengguna adblock plus dan adblock geta, filter ini gak berfungsi
> Tapi untuk pengguna ublock, adblockid plus sangat lancar

Sesuai penjelasan di [halaman depan](https://github.com/realodix/AdBlockID#cara-menggunakan), Saya hanya menyarankan pasang AdBlockID di uBlock Origin dan AdGuard, karena memang dibuat secara optimal di 2 produk itu saja.

---

from https://github.com/realodix/AdBlockID/discussions/1333#discussioncomment-4879070
> Situs layardrama.icu
> Ini walaupun pakai tamperkey gak bisa kebuka jika pakai adblock plus
> Kesimpulannya ini hanya untuk agar lebih efisien lagi jadi antara adblockid dan adblockid jadi lebih ringkas🙏🙏🙏

Mungkin yang Anda maksud https://layardrama21.icu? AdBlockID bekerja dengan baik, tidak ada pesan adblock terdeteksi.

Mungkin yang Anda maksud Tampermonkey? Saya tidak mengerti kenapa Anda menyebutkan Tampermonkey dan Adblock Plus? Saya tidak menemukan hubungan korelasi antara **manfaat penggabungan AdBlockID dan AdBlockID Plus** dengan **membuka situs https://layardrama21.icu pakai Tampermonkey dan Adblock Plus**.
